tests/foreman/ui/test_remoteexecution.py

```diff
@@ -624,26 +624,30 @@ def test_positive_configure_cloud_connector(
 
     template_name = 'Configure Cloud Connector'
     invocation_id = (
-        entities.JobInvocation().search(query={'search': f'description="{template_name}"'})[0].id
+        target_sat.api.JobInvocation()
+        .search(query={'search': f'description="{template_name}"'})[0]
+        .id
     )
     wait_for(
-        lambda: entities.JobInvocation(id=invocation_id).read().status_label
+        lambda: target_sat.api.JobInvocation(id=invocation_id).read().status_label
         in ["succeeded", "failed"],
         timeout="1500s",
     )
 
-    result = JobInvocation.get_output({'id': invocation_id, 'host': target_sat.hostname})
+    result = target_sat.cli.JobInvocation.get_output(
+        {'id': invocation_id, 'host': target_sat.hostname}
+    )
     logger.debug(f"Invocation output>>\n{result}\n<<End of invocation output")
     # if installation fails, it's often due to missing rhscl repo -> print enabled repos
     repolist = target_sat.execute('yum repolist')
     logger.debug(f"Repolist>>\n{repolist}\n<<End of repolist")
-
-    assert entities.JobInvocation(id=invocation_id).read().status == 0
-    assert 'project-receptor.satellite_receptor_installer' in result
+    assert target_sat.api.JobInvocation(id=invocation_id).read().status == 0
     assert 'Exit status: 0' in result
-    # check that there is one receptor conf file and it's only readable
-    # by the receptor user and root
-    result = target_sat.execute('stat /etc/receptor/*/receptor.conf --format "%a:%U"')
-    assert all(filestats == '400:foreman-proxy' for filestats in result.stdout.strip().split('\n'))
```

Comment:
There was once a big security issue with the conf file being readable by the users that shouldn't have access. I suggest checking that to avoid repeating this issue.

Reply:
I found no secrets in conf files of rhc, but I will talk with dev's more on this later.